bin/scalafmt

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env bash
```

Comment:
yeah olaf wants to deprecate the sbt-plugin and not everyone has scalafmt binary installed.